{
    "GSD": {
        "alias": "CVE-2022-1015",
        "description": "A flaw was found in the Linux kernel in linux/net/netfilter/nf_tables_api.c of the netfilter subsystem. This flaw allows a local user to cause an out-of-bounds write issue.",
        "id": "GSD-2022-1015",
        "references": [
            "https://advisories.mageia.org/CVE-2022-1015.html",
            "https://www.suse.com/security/cve/CVE-2022-1015.html",
            "https://ubuntu.com/security/CVE-2022-1015"
        ]
    },
    "namespaces": {
        "cve.org": {
            "CVE_data_meta": {
                "ASSIGNER": "secalert@redhat.com",
                "ID": "CVE-2022-1015",
                "STATE": "PUBLIC"
            },
            "affects": {
                "vendor": {
                    "vendor_data": [
                        {
                            "product": {
                                "product_data": [
                                    {
                                        "product_name": "kernel",
                                        "version": {
                                            "version_data": [
                                                {
                                                    "version_value": "Kernel 5.16.18"
                                                }
                                            ]
                                        }
                                    }
                                ]
                            },
                            "vendor_name": "n/a"
                        }
                    ]
                }
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "eng",
                        "value": "A flaw was found in the Linux kernel in linux/net/netfilter/nf_tables_api.c of the netfilter subsystem. This flaw allows a local user to cause an out-of-bounds write issue."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": [
                            {
                                "lang": "eng",
                                "value": "CWE-787"
                            }
                        ]
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://bugzilla.redhat.com/show_bug.cgi?id=2065323",
                        "refsource": "MISC",
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2065323"
                    },
                    {
                        "name": "http://blog.dbouman.nl/2022/04/02/How-The-Tables-Have-Turned-CVE-2022-1015-1016/",
                        "refsource": "MISC",
                        "url": "http://blog.dbouman.nl/2022/04/02/How-The-Tables-Have-Turned-CVE-2022-1015-1016/"
                    },
                    {
                        "name": "https://seclists.org/oss-sec/2022/q1/205",
                        "refsource": "MISC",
                        "url": "https://seclists.org/oss-sec/2022/q1/205"
                    }
                ]
            }
        },
        "nvd.nist.gov": {
            "configurations": {
                "CVE_data_version": "4.0",
                "nodes": []
            },
            "cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "secalert@redhat.com",
                    "ID": "CVE-2022-1015"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "A flaw was found in the Linux kernel in linux/net/netfilter/nf_tables_api.c of the netfilter subsystem. This flaw allows a local user to cause an out-of-bounds write issue."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": []
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://seclists.org/oss-sec/2022/q1/205",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://seclists.org/oss-sec/2022/q1/205"
                        },
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=2065323",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2065323"
                        },
                        {
                            "name": "http://blog.dbouman.nl/2022/04/02/How-The-Tables-Have-Turned-CVE-2022-1015-1016/",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "http://blog.dbouman.nl/2022/04/02/How-The-Tables-Have-Turned-CVE-2022-1015-1016/"
                        }
                    ]
                }
            },
            "impact": {},
            "lastModifiedDate": "2022-04-30T03:10Z",
            "publishedDate": "2022-04-29T16:15Z"
        }
    }
}